README.md

```diff
@@ -61,6 +61,27 @@ _otumat.DISABLE_USAGE_TRACKING_PACKAGES = (['datajoint'] +
 import datajoint
 ```
 
+## File Watching
+
+This feature allows you to run a given script when a selected file is modified.
+
+To watch a file, install `otumat` using `pip install otumat`, then run the command:
+  
+  `otumat watch [-h] -f WATCH_FILE [-i WATCH_INTERVAL] -s  WATCH_SCRIPT [watch_args ...]`
+
+### Arguments
+
+Help:
+ - `-h, --help`
+
+Required named arguments:
+ - `-f WATCH_FILE`: Path to file to be watched.
+ - `-s WATCH_SCRIPTT`: Path to script to run on file change.
+
+Optional named arguments:
```

Comment:
For these parameters that are optional, we should likely indicate what is the default.